<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>application</class>
 <widget class="QDialog" name="application">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>891</width>
     <height>651</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="createProfile">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>500</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Profile</string>
     </property>
    </widget>
    <widget class="QLabel" name="createProfileLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>581</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Create a Profile</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="mealsLineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>210</y>
       <width>450</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="ageLineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>420</y>
       <width>450</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="mainLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>599</width>
       <height>246</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>599</width>
       <height>246</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>599</width>
       <height>246</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Welcome to Quoala</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="exerciseLineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>280</y>
       <width>450</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="workLineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>350</y>
       <width>450</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="sleepLineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>450</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>460</y>
       <width>231</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Downloads/kaola-no-bg-2.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="feelingLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>501</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>How are you feeling?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="funBox">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>490</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Is it fun?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goodButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>130</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Good</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="taskLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>380</y>
       <width>461</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="whatYouLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>561</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>What do you have to do today?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="okayButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Okay</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addListButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>540</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add to List</string>
     </property>
    </widget>
    <widget class="QPushButton" name="doneButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>550</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
    <widget class="QPushButton" name="badButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Bad</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>220</y>
       <width>211</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Downloads/kaola-no-bg.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="listLabel">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>881</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>We recommend you to do your tasks in this order!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="listTask">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>661</width>
       <height>511</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="restartButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>580</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Goodbye!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>540</y>
       <width>221</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Downloads/kaola-no-bg-2.png</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
